### ISSUE TITLE
When updating from 0.999x to 1.0, the uninstall process says it's uninstalling 1.0

### ISSUE THREAD

**Issue by guyharris** [nmap/npcap#239]
Small cosmetic issue: I just updated Npcap on my Windows 10 VM, and noticed that the installer, when uninstalling the previous version of Npcap, which wasn't 1.0 (unless it got installed in the background today), it said it was uninstalling 1.0.

**Comment by dmiller-nmap** [comment]
Yes, this is a known issue. In order to avoid some bugs in previously-released uninstallers, for upgrades we unpack our own uninstaller and use that, provided the installed version is recent and compatible (IIRC this means 0.9985 and later). Unfortunately, NSIS uninstallers put the version number in the standard progress pages, and I haven't come up with a good way to override it.

**Comment by dmiller-nmap** [comment]
This was changed in nmap/npcap-build@2e4bb8fc627f41a3f57927e7a98d439d9974db03 and will be part of the next release. The uninstaller will retrieve the name and version it is uninstalling from the Registry, so it will correctly display OEM and version regardless of which version of the uninstaller is actually running. The install log (not deleted when upgrading) will show the internal version of the uninstaller that was executed.
